### ISSUE TITLE
login from coinvault is broken

### ISSUE THREAD

**Issue by dangershony** [Opdex/SOAP#5]
Login doesn't work anymore to stratisphere and I wonder if this PR is the reason
https://github.com/Opdex/SSAS/pull/4

This is the error I get
![image](https://user-images.githubusercontent.com/7487930/212670514-fab36ff6-0119-4ef3-abbf-8d23c5b9290d.png)

**Comment by bcExpt1123** [comment]
> Login doesn't work anymore to stratisphere and I wonder if this PR is the reason #4
> 
> This is the error I get ![image](https://user-images.githubusercontent.com/7487930/212670514-fab36ff6-0119-4ef3-abbf-8d23c5b9290d.png)

It works in the coinvault chrome extension, but doesn't work in the coinvault website.
The status code is 504/415.

**Comment by mrtpain** [comment]
I don't believe this is related to the package, there haven't been updates in a quite some time. Stratisphere has undergone some changes to wallet behavior which may have impacted how they handle requests on the server. 

504 gateway timeout, sounds like a stratisphere backend issue during verification process
